### DISCUSSION TITLE
Lost all (70+) items after upgrading to 2.6.0.301

### DISCUSSION THREAD

**Discussion by veneamin** [TumblThreeApp/TumblThree#305] [General]
Lost all (70+) items after upgrading to 2.6.0.301

**Desktop (please complete the following information):**
 - TumblThree version:  2.6.0.301 2.6.0.301
 - OS: Windows 10  pro 10.0.19044.0
 - Browser  Chrome
 - Version 100.0.4896.127

**Additional context**
After restart I got only 1 line in my tumbler list ( more than 75 items were before) 
Is there any backup config?

**Comment by thomas694** [comment]
Please provide us more details. Normally the blog list doesn't get lost on update.
Have you done the automatic update or manually? Are you using portable or normal mode? Has your download location changed in the settings?
Are there still files inside the Index subfolder of the download location? Is there something in a Backup or Archive subfolder?

**Comment by dajotim937** [comment]
Idk if it's related. I lost settings. there was file in app folder, but won't load. And app reset setting to default. When I ticked "activate portable mode" app rewrite setting file and I lost my old one. Used app's autoupdate.

**Comment by thomas694** [comment]
Unfortunately, due to an error, it could happen that the settings file was lost the first time the app was used after the update.

@veneamin + others
You should be able to 'recover' your blogs by setting your correct download location again. And check the settings once, if you changed other settings too.

**Comment by veneamin** [comment]
> Please provide us more details. Normally the blog list doesn't get lost on update. Have you done the automatic update or manually? 

App started and i saw message "New version aviable. Download and upgrade" - something like that. It download new version and automatically restarted without any errors. 

>Are you using portable or normal mode? 

Normal mode 

>Has your download location changed in the settings? 

Yes, it changed to abstract "Blogs", not my actual location
It seems that all settings  were set to defaults

>Are there still files inside the Index subfolder of the download location? 

All folders and files in download destination still there
I changed  destination in settings - and all blogs were restored!  :)

>Is there something in a Backup or Archive subfolder?

there are no such folders in app folder
And only "Archive" inside Index folder

**Comment by thomas694** [comment]
> If I will change destination in settings - all blogs will restored?

Yes, all blogs reappear and have unchanged blog settings. Only check the app settings once, if you used non-default settings.

> there are no such folders in app folder

These two folders can be inside the Index folder.

Unfortunately, this bug happened, but luckily it can be solved easily.

**Comment by thomas694** [comment]
For the documentation and to make it more clear.
The cause of this problem was solved in version 2.6.1.
If you skipped version 2.6.0 you are not affected.
Else you only need to set your `download location` again and check your app settings once.

**Comment by thomas694** [comment]
The issue has been closed. You can still comment. Feel free to ask for reopening the issue if needed.
